Library/Homebrew/abstract_command.rb

```diff
@@ -59,7 +64,15 @@ def initialize(argv = ARGV.freeze)
       @args = T.let(self.class.parser.parse(argv), CLI::Args)
     end
 
+    # This method will be invoked when the command is run.
+    #
+    # @api public
     sig { abstract.void }
     def run; end
   end
+
+  module Cmd
+    # The command class for `brew` itself, allowing its args to be parsed.
+    class Brew < AbstractCommand; end
+  end
```

Comment:
The addition of `Homebrew::Cmd::Brew` seems to have broken `brew typecheck --update` with:

```console
$ brew tc --update
Loading DSL extension classes... Done
Loading Rails application... Done
Loading DSL compiler classes... Done
Compiling DSL RBI files...

   identical  sorbet/rbi/dsl/homebrew/cmd/analytics.rbi
   identical  sorbet/rbi/dsl/homebrew/cmd/autoremove.rbi
Error: `Tapioca::Compilers::Args` failed to generate RBI for `Homebrew::Cmd::Brew`
   identical  sorbet/rbi/dsl/homebrew/cmd/cache.rbi
bundler: failed to load command: tapioca (/usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/bin/tapioca)
/usr/local/Homebrew/Library/Homebrew/sorbet/tapioca/compilers/args.rb:34:in `decorate': undefined method `name' for nil (NoMethodError)
	from /usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/tapioca-0.16.0/lib/tapioca/dsl/pipeline.rb:204:in `block in rbi_for_constant'
	from /usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/tapioca-0.16.0/lib/tapioca/dsl/pipeline.rb:200:in `each'
	from /usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/tapioca-0.16.0/lib/tapioca/dsl/pipeline.rb:200:in `rbi_for_constant'
	from /usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/tapioca-0.16.0/lib/tapioca/dsl/pipeline.rb:88:in `block in run'
	from /usr/local/Homebrew/Library/Homebrew/vendor/bundle/ruby/3.3.0/gems/parallel-1.26.2/lib/parallel.rb:650:in `call_with_index'
[...]
```

Reply:
Thanks for flagging. I’m AFK but can have a fix in the morning (PDT) if no one beats me to it.